source/reference/minio-server/settings/notifications/webhook-service.rst

```diff
@@ -272,6 +272,47 @@ Client Key
 
 Specify the path to the client private key to use for performing mTLS authentication to the webhook service.
 
+MaxRetry
```

Comment:
Do we know the default max number of retries or is it indefinite?

Reply:
Yeah, it's 0. Not sure why I neglected to add it. Fixed.